examples/test_suite_grpc/test_suite_grpc_client.py

```diff
@@ -0,0 +1,45 @@
+##########################################################################
+# Copyright (c) 2023-2023 Accenture
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+"""
+test_suite_1
+************
+:module: test_suite_1
+:synopsis: Showcase of grpc with pykiso
+"""
+
+import importlib
+import logging
+
+import pykiso
+from pykiso.auxiliaries import com_aux
+
+
+@pykiso.define_test_parameters(suite_id=1, case_id=1, aux_list=[com_aux])
+class DefaultTest(pykiso.BasicTest):
+    """With default parameters"""
+
+    def test_run(self):
+        answer = com_aux.send_message("")
+        assert answer == "Hello World!"
+
+
+@pykiso.define_test_parameters(suite_id=1, case_id=2, aux_list=[com_aux])
+class SpecificTest(pykiso.BasicTest):
+    """With specific parameters"""
+
+    def test_run(self):
+        answer = com_aux.send_message(
```

Comment:
How can this work? Our auxiliaries, including the CommunicationAuxiliary, don't accept any keyword args, so if we want to integrate this as a communication channel then we'll need to make all auxiliaries accept variable kwargs

I'm still somehow thinking that a GrpcClient would fit better with a simple auxiliary without cchannel

Reply:
Please take a look at the example now that it is working